bdb/bdb_verify.c

```diff
@@ -915,15 +915,16 @@ static int bdb_verify_ll(
                 }
 
             } else if (bdb_state->ixcollattr[ix]) {
-                if (dbt_data.size !=
-                    (sizeof(unsigned long long) + bdb_state->ixcollattr[ix])) {
+                if (dbt_data.size != (sizeof(unsigned long long) +
+                                      4 * bdb_state->ixcollattr[ix])) {
```

Comment:
I guess we wanted to store the number of decimal fields in a compound key. I agree that this needs to be cleaned up in the future.